docs/nidmm/session.rst

```diff
@@ -36,7 +36,7 @@ nidmm.Session
    +-----------------------------------------------------+----------------------------------------------+
    | :py:attr:`current_source`                           | float                                        |
    +-----------------------------------------------------+----------------------------------------------+
-   | :py:attr:`dc_bias`                                  | :py:data:`DCBias`                            |
+   | :py:attr:`dc_bias`                                  | int                                          |
```

Comment:
in attributes.rst this is listed as bool, same for some others.

Reply:
That's because it is one of the ones disconnected from the underlying enum. The values are ON (1) and OFF (0).

This is fixing the issue you brought up earlier in the PR that the type should be bool and not int. 

Which do you want?

Reply:
I'm a little confused.

Doesn't the client need to pass a bool for this property? Why would the documentation say int?

Reply:
Because there were multiple places this needed to be fixed. Added 'python_type' to attributes and used that everywhere instead of triplicating the logic.